File: article-api/src/main/kotlin/com/sns/article/reaction/infra/repository/SpringDataJdbcReactionRepository.java
package com.sns.article.reaction.infra.repository;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import com.sns.article.reaction.domain.Reaction;
import com.sns.article.reaction.domain.ReactionRepository;

@Repository
public interface SpringDataJdbcReactionRepository extends ReactionRepository, CrudRepository<Reaction, Long> {
}

File: article-api/src/main/kotlin/com/sns/article/reaction/infra/repository/SpringDataJdbcReactionRepository.kt
package com.sns.article.reaction.infra.repository

import com.sns.article.reaction.domain.Reaction
import com.sns.article.reaction.domain.ReactionRepository
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface SpringDataJdbcReactionRepository : ReactionRepository, CrudRepository<Reaction, Long>
